tests/test_compiler.py

```diff
@@ -50,6 +50,16 @@ def test_compile(project, contract):
     assert contract.source_id == f"{contract.name}.sol"
 
 
+@pytest.mark.benchmark(group="compile", timer=time.process_time)
+def test_compile_performance(benchmark, compiler, project):
+    """
+    See https://pytest-benchmark.readthedocs.io/en/latest/
+    """
+    source_path = project.contracts_folder / "MultipleDefinitions.sol"
+    result = benchmark(lambda: compiler.compile([source_path]))
```

Comment:
shouldn't this work?

```suggestion
    result = benchmark(compiler.compile, [source_path])
```